std_msgs/Header header
int32 dist
int32 strength
int32 temp